services/models/destination_rule.go

```diff
@@ -26,7 +48,7 @@ func (dRule *DestinationRule) Parse(destinationRule kubernetes.IstioObject) {
 	dRule.Name = destinationRule.GetObjectMeta().Name
 	dRule.CreatedAt = formatTime(destinationRule.GetObjectMeta().CreationTimestamp.Time)
 	dRule.ResourceVersion = destinationRule.GetObjectMeta().ResourceVersion
-	dRule.DestinationName = destinationRule.GetSpec()["name"]
+	dRule.Host = destinationRule.GetSpec()["host"]
```

Comment:
Mmm, did you rebase with master ?
In theory, these changes should be taken from the master, not fixed on these PR unless there is a bug.
Perhaps you would get some conflict then.

Reply:
Maybe was the change from istio-08 to master :'(